routes/bucket.routes.js

```diff
@@ -51,14 +52,9 @@ router.post("/create", isLoggedIn, async (req, res, next) => {
 router.get('/:bucketId/details', isLoggedIn, async (req, res) => {
     const { bucketId } = req.params
     try {
-        const bucket = await Bucket.findById(bucketId).populate('resources');
-        const currentUserId = getCurrentUser(req);
-
-        if(currentUserId === bucket.owner.toString()){
-            res.render('buckets/bucket-details', {bucket, active:'buckets'})
-        }else{
-            res.render('buckets/bucket-details-view', {bucket, active:'buckets'})
-        }
```

Comment:
You removed the logic for checking if the user is owner or not to show the details-view ?